java/j2ee.persistence/src/org/netbeans/modules/j2ee/persistence/editor/completion/ETCompletionContextResolver.java

```diff
@@ -105,12 +104,7 @@ private List<JPACompletionItem> completecreateNamedQueryparameters(JPACodeComple
         }
         if (ecs != null) {
             try {
-                entities = ecs.getEntityMappingsModel(false).runReadAction(new MetadataModelAction<EntityMappingsMetadata, Entity[]>() {
-                   @Override
-                    public Entity[] run(EntityMappingsMetadata metadata) throws Exception {
-                        return metadata.getRoot().getEntity();
-                    }
-                });
+                entities = ecs.getEntityMappingsModel(false).runReadAction( (EntityMappingsMetadata metadata) -> metadata.getRoot().getEntity() );
```

Comment:
just a nitpick. Consider the one-action-per-line formatting when method chaining is used. Makes the lines shorter.
```java
entities = ecs.getEntityMappingsModel(false)
              .runReadAction( (EntityMappingsMetadata metadata) -> metadata.getRoot().getEntity() );
```
there are many comparable occurrences like this one

Reply:
I like it that way too. Done

Reply:
@pepness there are some more invocations of this method in the same style which you converted to lambdas btw. You changed only the one occurrence so far if I see this correctly.

search for `getEntityMappingsModel(false).` in
https://patch-diff.githubusercontent.com/raw/apache/netbeans/pull/5444.diff